src/shared/ContentsTable/utils/adjustListIfUpDir.tsx

```diff
@@ -33,8 +33,12 @@ export function adjustListIfUpDir({
   displayType: 'LIST' | 'TREE'
   rawTableRows: Row[]
 }) {
-  if (treePaths.length > 1 && displayType === displayTypeParameter.tree) {
-    const upDir = treePaths?.at(-2)
+  if (
+    Array.isArray(treePaths) &&
+    treePaths.length > 1 &&
```

Comment:
if treePaths is a string or is an object with a length key this would fail previously. Just adding an isArray check